modules/sandboxed-containers-deploying-osc-with-other-ocp-components.adoc

```diff
@@ -0,0 +1,16 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/understanding_sandboxed_containers.adoc
+
+:_content-type: CONCEPT
+[id="sandboxed-containers-with-other-ocp-components_{context}"]
+
+= Using {sandboxed-containers-first} with {VirtProductName}
+//= Using {sandboxed-containers-first} with other {product-title} components
+
+You can use {sandboxed-containers-first} on clusters installed with {VirtProductName}.
+
+You must configure the following on {VirtProductName} to enable using {sandboxed-containers-first}:
```

Comment:
Configuring those parameters does not enable one to run sandboxed contianers, that can be done anyways. 

Maybe something along the lines of

To ensure compatibility when running Virt + sanboxed containers at the same time, configure the following virt parameters:...